src/components/ContactPolicy/ContactPolicy.js

```diff
@@ -1,20 +1,22 @@
 import React, { useState } from 'react';
 import { Helmet } from 'react-helmet';
-import { useIntl } from 'react-intl';
+import { FormattedMessage, useIntl } from 'react-intl';
 import HeaderSection from '../shared/HeaderSection/HeaderSection';
 import { Breadcrumb, BreadcrumbItem } from '../shared/Breadcrumb/Breadcrumb';
 import { FormattedMessageMarkdown } from '../../i18n/FormattedMessageMarkdown';
 import { Switch } from '../shared/Switch/Switch';
 import { FieldGroup, FieldItem, NumberField, SubmitButton } from '../form-helpers/form-helpers';
 import { Form, Formik } from 'formik';
 import useTimeout from '../../hooks/useTimeout';
+import { InjectAppServices } from '../../services/pure-di';
 
-export const ContactPolicy = () => {
-  const [active, toggleActive] = useState(false);
-  const [formSubmitted, setFormSubmitted] = useState(false);
+export const ContactPolicy = InjectAppServices(({ dependencies: { experimentalFeatures } }) => {
+  const [isActive, setIsActive] = useState(false);
+  const [isFormSubmitted, setIsFormSubmitted] = useState(false);
```

Comment:
@jcamposmk @lucianaDilizio @aferraromedina opinions on this?

Reply:
https://github.com/FromDoppler/doppler-webapp/pull/1461#discussion_r663162841

Reply:
Estuvimos discutiendo sobre este tema, porque @jcamposmk basandosé en _Clean Code_ propone usar siempre el prefijo `is` para los booleanos.

Pero en general, no es lo que estamos haciendo.

Así que por ahora preferimos que si está claro que es un estado (formas infinitivas o participio, `loading`, `closed`, `active`), no usar el `is`. En casos donde puede haber confusiones, sustantivos o verbos, ahí si, por ejemplo `isError` o `isOpen`.

Reply:
The same here
```suggestion
  const [formSubmitted, setFormSubmitted] = useState(false);
```